web-client/src/routerPublic.js

```diff
@@ -80,6 +85,11 @@ const router = {
       )();
     });
 
+    route('/maintenance', () => {
+      setPageTitle('Maintenance');
+      return app.getSequence('gotoMaintenanceSequence')();
+    });
+
```

Comment:
Doesn't seem to be hurting anything, but I don't think we want this in duplicate. do we?